xrally_kubernetes/tasks/scenarios/pods.py

```diff
@@ -0,0 +1,118 @@
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import time
+
+from rally.task import scenario
+
+from xrally_kubernetes.tasks import scenario as common_scenario
+
+
+@scenario.configure(name="Kubernetes.create_and_delete_pod",
+                    platform="kubernetes")
+class CreateAndDeletePod(common_scenario.BaseKubernetesScenario):
+
+    def _parse_pod_status_conditions(self, conditions):
+        """Method for collecting pods statuses: inited, scheduled, ready."""
+
+        def to_time(dt):
+            return time.mktime(dt)
+
+        if not conditions:
+            return []
+
+        start, finish = None, None
+        stat_dict = {
```

Comment:
using dict her and stat_dict.items() at L67 makes the order of the legend in the final report unstable. it would be nice to save the correct order of statuses. there are 2 options: use `collections.OrderedDict([("kubernetes.initialized_pod", {}), ("kubernetes.scheduled_pod", {})])` or separate variables for each state.